node/setup.go

```diff
@@ -280,7 +280,7 @@ func createMempoolAndMempoolReactor(
 	switch config.Mempool.Type {
 	// allow empty string for backward compatibility
 	case cfg.MempoolTypeFlood, "":
-		lanesInfo, err := mempl.BuildLanesInfo(appInfoResponse.LanePriorities, types.Lane(appInfoResponse.DefaultLanePriority))
+		lanesInfo, err := mempl.BuildLanesInfo(appInfoResponse.LanePriorities, types.LaneID(appInfoResponse.DefaultLane))
```

Comment:
I think `types.LaneID` should not be here. The second parameter to `BuildLanesInfo` should be changed to `string`.